packages/bridge-status-controller/tsconfig.json

```diff
@@ -11,7 +11,11 @@
     { "path": "../controller-utils" },
     { "path": "../network-controller" },
     { "path": "../polling-controller" },
-    { "path": "../transaction-controller" }
+    { "path": "../assets-controllers" },
+    { "path": "../transaction-controller" },
+    { "path": "../gas-fee-controller" },
+    { "path": "../preferences-controller" },
```

Comment:
Nit: I don't see where preferences-controller is being used in this controller ? can we remove this ?